include/mbgl/gl/gl_values.hpp

```diff
@@ -76,7 +76,7 @@ struct DepthMask {
     }
 };
 
-struct ColorMask {
+struct ColorMaskValue {
```

Comment:
The goal of these values, and their surrounding `Value` objects is to prevent duplicate state from being set. It seems like this is no longer the case with this code? Will it come back as work progresses?